ON_CHAIN_UPDATES.md

```diff
@@ -18,3 +18,11 @@ To update this document, follow the steps below:
 - 2025-03-26: Updated `Native Syndicate Cover` cover wording.
 
 ## Listings Updates
+
+- 2025-04-01: Updated `Elite Cover` updating the Annex and the UI copy for this listing.
+- 2025-04-01: Updated `Beefy` updating the Annex and the UI copy for this listing.
+- 2025-04-01: Renamed `Morpho + Morpho Vaults` to `Blue Chip Morpho Vaults`. We're renaming this existing listing and updating the Annex.
+- 2025-04-01: Renamed `Euler Finance v2` to `Blue Chip Euler v2 Markets`. We're renaming this existing listing and updating the Annex.
+- 2025-04-01: Added new listing `B-Tier Morpho Vaults`.
+- 2025-04-01: Added new listing `B-Tier Euler v2 Markets`.
+- 2025-04-01: Added new listing `OC Cover`. `Private` listing.
```

Comment:
maybe I misunderstood it, but I thought we'll only edit this file for dummy commits to force a version bump and publish a patch version of the sdk